Sources/BraveTalk/BraveTalkJitsiCoordinator.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import JitsiMeetSDK
+
+/// Handles coordinating when to use the Jitsi SDK for better Brave Talk integration
+///
+/// Currently relies on AppConstants.buildChannel
+@MainActor public class BraveTalkJitsiCoordinator: Sendable {
+  /// Whether or not the jitsi SDK integration is enabled
+  static var isIntegrationEnabled: Bool {
+    return AppConstants.buildChannel != .release
```

Comment:
Do we want to have it in public beta? if not, have to change to `!AppConstants.buildChannel.isPublic`

Reply:
Yeah they want it in public betas